composer.json

```diff
@@ -39,41 +39,41 @@
         "pagerfanta/pagerfanta": "^1.0.4",
         "piwik/device-detector": "^3.7",
         "ramsey/uuid": "^3.1",
-        "sensio/framework-extra-bundle": "^3.0",
+        "sensio/framework-extra-bundle": "^5.0",
```

Comment:
@danrot @chirimoya oriented by [symfony-standard](https://github.com/symfony/symfony-standard/blob/3.4/composer.json#L22) updated the framework bundle to 5.0